.pre-commit-config.yaml

```diff
@@ -29,4 +29,6 @@ repos:
       - id: ruff # replaces Flake8, isort, pydocstyle, pyupgrade
         args:
           - --fix
+          - --ignore=E402
```

Comment:
Would this be as effective in this section of `pyproject.toml`?
https://github.com/prjemian/model_instrument/blob/57649763eaa225d859d84f24352d6d5d608ecbf0/pyproject.toml#L187-L189